components/brave_vpn/resources/panel/components/settings-panel/index.tsx

```diff
@@ -26,64 +28,90 @@ function SettingsPanel (props: Props) {
   const handleClick = (entry: string) => {
     getPanelBrowserAPI().panelHandler.openVpnUI(entry)
   }
-  const handleKeyDown = (entry: string, event: React.KeyboardEvent<HTMLAnchorElement>) => {
-    if (event.keyCode !== 32) { return }
+  const handleKeyDown = (
+    entry: string,
+    event: React.KeyboardEvent<HTMLAnchorElement>
+  ) => {
+    if (event.keyCode !== 32) {
+      return
+    }
     if (entry === 'support') {
       props.showContactSupport()
       return
     }
     handleClick(entry)
   }
-  const handleToggleChange = (isOn: boolean) => {
-    getPanelBrowserAPI().serviceHandler.enableOnDemand(isOn)
+
+  const handleToggleChange = ({ checked }: { checked: boolean }) => {
+    getPanelBrowserAPI().serviceHandler.enableOnDemand(checked)
   }
 
   return (
     <S.Box>
       <S.PanelContent>
         <S.PanelHeader>
-          <S.BackButton
+          <S.StyledButton
             type='button'
             onClick={props.closeSettingsPanel}
             aria-label='Close settings'
           >
-            <i><CaratStrongLeftIcon /></i>
-            <span>{getLocale('braveVpnSettingsPanelHeader')}</span>
-          </S.BackButton>
+            <S.StyledIcon name='arrow-left'></S.StyledIcon>
+          </S.StyledButton>
+          <S.HeaderLabel>
+            {getLocale('braveVpnSettingsPanelHeader')}
+          </S.HeaderLabel>
         </S.PanelHeader>
-        <S.List>
-          {onDemand.available && <li>
-            <S.ReconnectBox>
-              <span>
+        <S.SettingsList>
+          {onDemand.available && (
+            <S.Setting>
+              <S.SettingLabel>
                 {getLocale('braveVpnReconnectAutomatically')}
-              </span>
+              </S.SettingLabel>
               <Toggle
-                isOn={onDemand.enabled}
+                checked={onDemand.enabled}
                 onChange={handleToggleChange}
-                brand='vpn'
-                size='sm'
+                size='small'
                 aria-label='Reconnect automatically'
               />
-            </S.ReconnectBox>
-          </li>}
-          <li>
-            <a href="#" onClick={handleClick.bind(this, 'manage')} onKeyDown={handleKeyDown.bind(this, 'manage')}>
-              {getLocale('braveVpnManageSubscription')}
+            </S.Setting>
+          )}
+          <S.Setting>
+            <a
+              href='#'
+              onClick={handleClick.bind(this, 'manage')}
+              onKeyDown={handleKeyDown.bind(this, 'manage')}
+            >
```

Comment:
optional nit:

I know this wasn't you but this is pretty weird - I guess it works but I think more idiomatic would be:

```suggestion
              onClick={e => handleClick('manage', e)}
              onKeyDown={e => handleKeyDown('manage', e)}
            >
```

optional for this and others in the fiel